src/main/kotlin/no/nav/familie/ba/sak/kjerne/vedtak/begrunnelser/EØSStandardbegrunnelse.kt

```diff
@@ -91,6 +91,38 @@ enum class EØSStandardbegrunnelse : IVedtakBegrunnelse {
         override val sanityApiNavn = "innvilgetSekundaerlandHardKodet"
         override val vedtakBegrunnelseType = VedtakBegrunnelseType.EØS_INNVILGET
     },
+    INNVILGET_SEKUNDÆRLAND_ALENEANSVAR {
+        override val sanityApiNavn = "innvilgetSekundaerlandAleneansvar"
+        override val vedtakBegrunnelseType = VedtakBegrunnelseType.EØS_INNVILGET
+    },
+    INNVILGET_TILLEGGSTEKST_NULLUTBETALING {
+        override val sanityApiNavn = "innvilgetTilleggstekstNullutbetaling"
+        override val vedtakBegrunnelseType = VedtakBegrunnelseType.EØS_INNVILGET
+    },
+    INNVILGET_SEKUNDÆRLAND_UK_STANDARD {
+        override val sanityApiNavn = "innvilgetSekundaerlandUkStandard"
+        override val vedtakBegrunnelseType = VedtakBegrunnelseType.EØS_INNVILGET
+    },
+    INNVILGET_SEKUNDÆRLAND_UK_ALENEANSVAR {
+        override val sanityApiNavn = "innvilgetSekundaerlandUkAleneansvar"
+        override val vedtakBegrunnelseType = VedtakBegrunnelseType.EØS_INNVILGET
+    },
+    INNVILGET_SEKUNDÆRLAND_UK_OG_UTLAND_STADARD {
+        override val sanityApiNavn = "innvilgetSekundaerlandUkOgUtlandStandard"
+        override val vedtakBegrunnelseType = VedtakBegrunnelseType.EØS_INNVILGET
+    },
+    INNVILGET_SEKUNDAERLAND_TO_ARBEIDSLAND_NORGE_UTBETALER {
```

Comment:
```suggestion
    INNVILGET_SEKUNDÆRLAND_TO_ARBEIDSLAND_NORGE_UTBETALER {
```
Småpirk, men sikkert greit å være konsistent om vi bruker `SEKUNDAERLAND` eller `SEKUNDÆRLAND` som enum verdi.

Reply:
Det samme gjelder linje 118.

Reply:
Fikset i https://github.com/navikt/familie-ba-sak/pull/2808/commits/3fe5b3aa7bd60f16963a6f79d1c3ddb0673a8224